tests/test_train.py

```diff
@@ -129,3 +130,22 @@ def test_train_nlu_temp_files(
     )
 
     assert count_temp_rasa_files(tempfile.tempdir) == 0
+
+
+def test_train_nlu_wrong_format_error_message(
+    capsys: CaptureFixture,
+    tmp_path: Text,
+    monkeypatch: MonkeyPatch,
+    default_stack_config: Text,
+    incorrect_nlu_data: Text,
+):
+    monkeypatch.setattr(tempfile, "tempdir", tmp_path)
+
+    train_nlu(
+        default_stack_config,
+        incorrect_nlu_data,
+        output="test_train_nlu_temp_files_models",
+    )
+
+    captured = capsys.readouterr()
+    assert "Please verify the data format" in captured.out
```

Comment:
should we also test that the actual training is not running?
And is there a test for the `No NLU data given.` case?

Reply:
that's a very good point! Yes, the situation with `No NLU data given` can still happen imo. 
I've added one more tests case!